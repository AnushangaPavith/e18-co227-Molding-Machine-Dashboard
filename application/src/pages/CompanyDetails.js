import React from 'react';
import Boxes from '@material-ui/core/Box';
import { BsFillPeopleFill } from "react-icons/bs";
import { BsFillInfoCircleFill } from "react-icons/bs";
import { FaBuilding } from "react-icons/fa";
import { BsPersonPlusFill } from "react-icons/bs";


function Company() {

    return (
        <div className="company-container"> 

            <Boxes className='company-box'>
                About Us <br/>
                <BsFillInfoCircleFill size={70} role='svg' />
            </Boxes>

            <Boxes className='company-box'>
                People <br/>
                <BsFillPeopleFill size={70} role='svg' />
            </Boxes>

            <Boxes className='company-box'>
                Facilities <br/>
                <FaBuilding size={70} role='svg' />
            </Boxes>

            <Boxes className='company-box'>
                Join us <br/>
                <BsPersonPlusFill size={70} role='svg' />
            </Boxes>
      
        </div>
        
    )

}



export default Company;