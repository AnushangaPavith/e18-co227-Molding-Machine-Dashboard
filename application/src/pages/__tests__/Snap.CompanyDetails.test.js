import { render, cleanup } from "@testing-library/react";
import {BrowserRouter as Router} from 'react-router-dom';
import CompanyDetails from "../CompanyDetails";

afterEach(() => { cleanup(); })

test("render and take snapshot of company details page", async () => {
    const { asFragment } = render(
        <Router>
            <CompanyDetails />
        </Router>);

    expect(asFragment()).toMatchSnapshot();
})
