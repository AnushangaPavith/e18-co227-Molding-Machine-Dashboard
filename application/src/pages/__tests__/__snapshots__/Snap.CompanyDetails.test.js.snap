// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render and take snapshot of company details page 1`] = `
<DocumentFragment>
  <div
    class="company-container"
  >
    <div
      class="MuiBox-root MuiBox-root-1 company-box"
    >
      About Us 
      <br />
      <svg
        fill="currentColor"
        height="70"
        role="svg"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 16 16"
        width="70"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M8 16A8 8 0 1 0 8 0a8 8 0 0 0 0 16zm.93-9.412-1 4.705c-.07.34.029.533.304.533.194 0 .487-.07.686-.246l-.088.416c-.287.346-.92.598-1.465.598-.703 0-1.002-.422-.808-1.319l.738-3.468c.064-.293.006-.399-.287-.47l-.451-.081.082-.381 2.29-.287zM8 5.5a1 1 0 1 1 0-2 1 1 0 0 1 0 2z"
        />
      </svg>
    </div>
    <div
      class="MuiBox-root MuiBox-root-2 company-box"
    >
      People 
      <br />
      <svg
        fill="currentColor"
        height="70"
        role="svg"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 16 16"
        width="70"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M7 14s-1 0-1-1 1-4 5-4 5 3 5 4-1 1-1 1H7zm4-6a3 3 0 1 0 0-6 3 3 0 0 0 0 6z"
        />
        <path
          d="M5.216 14A2.238 2.238 0 0 1 5 13c0-1.355.68-2.75 1.936-3.72A6.325 6.325 0 0 0 5 9c-4 0-5 3-5 4s1 1 1 1h4.216z"
          fill-rule="evenodd"
        />
        <path
          d="M4.5 8a2.5 2.5 0 1 0 0-5 2.5 2.5 0 0 0 0 5z"
        />
      </svg>
    </div>
    <div
      class="MuiBox-root MuiBox-root-3 company-box"
    >
      Facilities 
      <br />
      <svg
        fill="currentColor"
        height="70"
        role="svg"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="70"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M436 480h-20V24c0-13.255-10.745-24-24-24H56C42.745 0 32 10.745 32 24v456H12c-6.627 0-12 5.373-12 12v20h448v-20c0-6.627-5.373-12-12-12zM128 76c0-6.627 5.373-12 12-12h40c6.627 0 12 5.373 12 12v40c0 6.627-5.373 12-12 12h-40c-6.627 0-12-5.373-12-12V76zm0 96c0-6.627 5.373-12 12-12h40c6.627 0 12 5.373 12 12v40c0 6.627-5.373 12-12 12h-40c-6.627 0-12-5.373-12-12v-40zm52 148h-40c-6.627 0-12-5.373-12-12v-40c0-6.627 5.373-12 12-12h40c6.627 0 12 5.373 12 12v40c0 6.627-5.373 12-12 12zm76 160h-64v-84c0-6.627 5.373-12 12-12h40c6.627 0 12 5.373 12 12v84zm64-172c0 6.627-5.373 12-12 12h-40c-6.627 0-12-5.373-12-12v-40c0-6.627 5.373-12 12-12h40c6.627 0 12 5.373 12 12v40zm0-96c0 6.627-5.373 12-12 12h-40c-6.627 0-12-5.373-12-12v-40c0-6.627 5.373-12 12-12h40c6.627 0 12 5.373 12 12v40zm0-96c0 6.627-5.373 12-12 12h-40c-6.627 0-12-5.373-12-12V76c0-6.627 5.373-12 12-12h40c6.627 0 12 5.373 12 12v40z"
        />
      </svg>
    </div>
    <div
      class="MuiBox-root MuiBox-root-4 company-box"
    >
      Join us 
      <br />
      <svg
        fill="currentColor"
        height="70"
        role="svg"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 16 16"
        width="70"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M1 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1H1zm5-6a3 3 0 1 0 0-6 3 3 0 0 0 0 6z"
        />
        <path
          d="M13.5 5a.5.5 0 0 1 .5.5V7h1.5a.5.5 0 0 1 0 1H14v1.5a.5.5 0 0 1-1 0V8h-1.5a.5.5 0 0 1 0-1H13V5.5a.5.5 0 0 1 .5-.5z"
          fill-rule="evenodd"
        />
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
